config/locales/forms/zh.yml

```diff
@@ -109,16 +109,15 @@ zh:
       nickname: 设备昵称
       nickname_hint: 这样如果你添加了更多使用人脸或触摸解锁的设备的话，你就能把它们分辨开来。
     webauthn_setup:
-      intro: Use your physical security key to add an additional layer of protection
-        to your %{app_name} account to prevent unauthorized access.
+      intro: 使用实体安全密钥为您的 %{app_name} 帐户添加了额外一层安全保障，以防止未经授权的访问。了解有关安全密钥的更多信息
       learn_more: Learn more about security keys
       nickname: 安全密钥昵称
       saving: 正在保存你的凭据…
-      set_up: Set up security key
+      set_up: 设置安全密钥
       step_1: Give it a nickname
```

Comment:
Do we know if these remaining strings are accounted for in another ticket?

Reply:
I just pinged the team to let them know. its in the acceptance for this ticket.